_data/simple-icons.json

```diff
@@ -4514,7 +4517,7 @@
         {
             "title": "Jamstack",
             "hex": "F0047F",
-            "source": "https://github.com/jamstack/jamstack.org/tree/main/src/site/img/logo"
+            "source": "https://github.com/jamstack/jamstack.org/blob/main/src/site/img/logo/svg/Jamstack_Icon_Original.svg"
```

Comment:
Should be https://github.com/jamstack/jamstack.org/blob/a7de230798f98bdde78f0a0eeb5ebfc488c563aa/src/site/img/logo/svg/Jamstack_Icon_Original.svg